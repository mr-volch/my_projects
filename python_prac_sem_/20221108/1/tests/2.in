b = DivStr("GeberoPerandoMerkatoLehanto")
print(list(b//2))
print(b%13)
print(b%18)
print((list(b//3)))
