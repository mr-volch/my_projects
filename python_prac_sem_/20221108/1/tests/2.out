['GeberoPerando', 'MerkatoLehant']
o
toLehanto
['GeberoPer', 'andoMerka', 'toLehanto']
